docs/modules/ROOT/pages/how-tos/worklows/git-submodules.adoc

```diff
@@ -0,0 +1,90 @@
+= Git Submodule pattern
```

Comment:
You are introducing the term "pattern" ... can you explain what you mean by that? Or give background for what this pattern solves?

Reply:
I'm not sure I want to use "pattern" anymore. Do you have any recommendations? Here are some ideas replacements I can think of:
- Guideline
- Approach
- Protocol
- Blueprint

For now I swapped it to "guideline".

I remember in our discussion that we want to emphasize that this is **not** officially supported by Konflux, but I'm still not sure what word would be good...